tmp/tmp.md

```diff
@@ -0,0 +1 @@
+Place to store temporary file
```

Comment:
Similarly, a tmp directory for the user should not be committed to the project.

Reply:
Deleted